### PR TITLE
[restinio] update to 0.6.2

### PR DESCRIPTION
**Pull request by eao197**
RESTinio updated to the latest version.

### CODE REVIEW

**Review by LilyWangL — CHANGES_REQUESTED** [review]
`include(vcpkg_common_functions)` is deprecated. Please remove it. 
Thanks.

File: ports/restinio/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: restinio
-Version: 0.6.1.1
+Version: 0.6.2
 Description: A header-only C++14 library that gives you an embedded HTTP/Websocket server targeted primarily for asynchronous processing of HTTP-requests.
```
Comment (LilyWangL) [inline ports/restinio/CONTROL]: Can you please add a **Homepage: https://github.com/stiffstream/restinio** for this port?

**Review by LilyWangL — CHANGES_REQUESTED** [review]

File: ports/restinio/CONTROL (hunk 1 above)
Comment (eao197) [reply ports/restinio/CONTROL]: Added.

**Comment by eao197** [comment]
> include(vcpkg_common_functions) is deprecated. Please remove it.

Removed.

**Review by LilyWangL — APPROVED** [review]

**Review by dan-shaw — APPROVED** [review]

**Pull request merged**